README.md

```diff
@@ -51,6 +51,24 @@ of variables:
 In addition, you will need to ensure the bucket CORS `AllowedOrigin` settings
 are configured to allow for the domain this app runs at.
 
+## Sending Receipt Email in Production
+
+`SMTP_ADDRESS`
+`SMTP_PASSWORD`
+`SMTP_PORT`
+`SMTP_USER`
+`THESIS_ADMIN_EMAIL` - used for both `from` and `cc` of receipt emails.
+
+In development, emails are written to a file in `tmp`. In testing, they are
+stored in memory. You still need the `THESIS_ADMIN_EMAIL` set for the tmp file
+to be written without errors.
+
+On staging, the default is disabled email. Unset `DISABLE_ALL_EMAIL` if you
+have a reason to turn them on. Due to the potential of unwanted emails being
+sent when `FAKE_AUTH_ENABLED` is enabled (like on PR builds), it's best to
```

Comment:
Should we make the fake auth setting override the disable all email setting, just in case?

Reply:
I don't think so. On PR builds you'd have to make several intentional choices to send email.